File: cap.10_linguagem_c++/codigos/exercicios/MENOR_DE_TRES.cpp
#include <iostream>

using namespace std;

int main() {
    int num1, num2, num3 = 0;

    cout << "Primeiro valor: ";
    cin >> num1;
    cout << "Segundo valor: ";
    cin >> num2;
    cout << "Terceiro valor: ";
    cin >> num3;

    if ((num1 <= num2) && (num1 <= num3)) {
        cout << "MENOR = " << num1;
    } else if (num2 <= num3) {
        cout << "MENOR = " << num2;
    } else {
        cout << "MENOR = " << num3;
    }

    return 0;
}

File: cap.11_java/codigos/exercicios/src/MenorDeTres.java
import java.util.Locale;
import java.util.Scanner;

public class MenorDeTres {

    public static void main(String[] args) {
        
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int val1, val2, val3 = 0;

        System.out.print("Primeiro valor: ");
        val1 = sc.nextInt();
        System.out.print("Segundo valor: ");
        val2 = sc.nextInt();
        System.out.print("Terceiro valor: ");
        val3 = sc.nextInt();

        if (val1 <= val2 && val1 <= val3) {
            System.out.println("MENOR = " + val1);
        } else if (val2 <= val3) {
            System.out.println("MENOR = " + val2);
        } else {
            System.out.println("MENOR = " + val3);
        }

        sc.close();
    }

}
